source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+AppConfig
+---------
+
+“A community focused on providing tools and best practices around native capabilities in mobile operating systems to enable a more consistent, open and simple way to configure and secure mobile apps in order to increase mobile adoption in business.” - AppConfig.org
+
+So that sounds great, but are there any benefits for users?
+
+Again, in the AppConfig Community’s words, “Users benefit with instant mobile productivity and a seamless out-of-the box experience, and businesses benefit with secure work-ready apps with minimal setup required while leveraging existing investments in Enterprise Mobility Management (EMM), VPN, and identity solutions. Put another way, your apps are simpler to configure, secure and deploy.”
+
+For now, focus on that last part, “... your apps are simpler to configure, secure and deploy.” AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
+
+Application (App) Wrapping
+--------------------------
+From the article we `referenced earlier <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_, application(app) wrapping involves “...the use of an SDK from an EMM vendor that allows a developer or admin to deploy an API that enables management policies to be set up.” 
+
+When going this route, there are two options:
+
+**Option 1**
+    The EMM provider gives you their libraries, and then you go to the source code for the app. Using the libraries you "wrap" the source and repackage the application. This approach will take significant development time and associated frustration. In the end you should have an app that is now wrapped with the EMM libraries. The hope, but not the guarantee, is that you have an app with an additional layer. This allows you to manage and secure the app on a user’s mobile device.
+
+**Option 2**
+    The EMM provider gives you a tool used for wrapping the app. In more modern cases, it’s a simple checkbox in your EMM application when configuring the app. These tools then inject all the needed code to wrap it. Then the tool (or EMM provider) builds it back up. This allows you to distribute the new, wrapped app. The wrapped app has a layer allowing you to manage and secure the app on a user’s mobile device. 
+
+Until recently app wrapping has been the common approach. It does not come without risks and challenges to scalability though. 
+
+For example, most modern applications follow continuous development. Each time an application changes it will need to go through the process described above. 
+
+There is also functionality and compatibility risk. This is a known issue for the Mattermost application and app wrapping tools.
```

Comment:
Personally, I think this is where we should have the detail (that is currently later in the article) that explains the issue with React Native. This section goes into a lot of detail for something we don't support but for me misses off the key bit of explanation. The later part of the article for me should be more focused on guidance on what to do, not the detail of why we don't support it.

Reply:
I’m 3/5 with either keeping it as is, or having it in both places for redundancy of the message.

Reply:
Keep as is, and we'll ponder this for v2. 5/5